.github/workflows/testing.yml

```diff
@@ -18,7 +18,7 @@ jobs:
   example_tests:
     name: Example tests
     runs-on: ubuntu-latest
-    container: python:3.8-slim
+    container: python:3.8
```

Comment:
Using `python:3.8-slim` crashed the action that uploads the plots to the `examples-material` branch.

But I this should be fine here I guess?